Status command

otter-computer opened the pull request:
Adds the `!Status`command.
- Displays the time the bot was deployed
- Shows the latest commit
- Has a link to the commit page on GitHub
- Only useable by members with the `Bot Developer` role

otter-computer committed:
Merge pull request #1 from gaymers-discord/master

Add !status command for testing

otter-computer committed:
remove status command

otter-computer committed:
Add status command

otter-computer committed:
fuck off github you piece of absolute shite

otter-computer committed:
Merge branch 'gaymers-discord-master'

otter-computer committed:
Merge pull request #155 from ThomasShaped/master

status test

otter-computer committed:
Changed command name from reserved word

'status' could possibly be a reserved word, don't wanna take that risk...

otter-computer committed:
Update status message formatting

otter-computer committed:
Changed my mind, it seemed to work before.

otter-computer committed:
Change usage text

jmariondev commented on commands/status.js:
```diff
@@ -0,0 +1,13 @@
+  process: (bot, message) => {
+    message.reply(
+      'Bot deployed at: `' + process.env.HEROKU_RELEASE_CREATED_AT + '`\n' +
+      'https://github.com/gaymers-discord/DiscoBot/commit/' + process.env.HEROKU_SLUG_COMMIT + '\n' +
```
Maybe wrap on 80 here?

jmariondev reviewed (changes requested):
Maybe wrap on 80 with this code?

Also, it would be good to integrate #157 into this so the command can only be used in the bot dev room.

Other than that, LGTM.

otter-computer committed:
Added onlyIn

- Added onlyIn prop
- Truncated message lines to 80 (ish, I did this on github web editor don't hurt me)
